code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -445,3 +445,69 @@
 		M.emote(pick("twitch","laugh","frown"))
 	..()
 	. = 1
+
+/datum/reagent/drug/synthetic_cocaine
+	name = "Synthetic Cocaine"
+	id = "synthetic_cocaine"
+	description = "Reduces stun times and increases movement speed, as well as stopping movement slowdown from damage. Highly addictive."
+	reagent_state = LIQUID
+	color = "#E6FFFF"
+	addiction_threshold = 10
+	overdose_threshold = 20
+
+/datum/reagent/drug/synthetic_cocaine/on_mob_life(mob/living/M)
+	var/high_message = pick("You feel like you can take on the world.", "You feel like you can walk along the razor's edge.", "You feel like you can push it to the limit.")
+	if(prob(5))
+		to_chat(M, "<span class='notice'>[high_message]</span>")
+	M.AdjustStun(-20, FALSE)
+	M.AdjustKnockdown(-20, FALSE)
```

Comment:
This is inconsisent with all other stun reduction chems, where all 5 reduction types have the same value.